client/src/pages/Stories/Stories.css

```diff
@@ -0,0 +1,17 @@
+h1 {
+    margin-top: 5%;
+    text-align: center;
+}
+
+li {
+    align-items: center;
+    display: inline-block;
+    justify-content: space-evenly;
+    vertical-align: top;
+    width: 33%;
+}
+
+p {
+    margin-left: 5%;
+    margin-right: 5%;
+}
```

Comment:
Rather than adding styles to the various html elements, can you add styling to classes and then give the html elements those classes?